### PR TITLE
Load Existing Data Import Rows in GAU Widget

### PR DESCRIPTION
**Pull request by dfuller114**
# Critical Changes

# Changes
When a data import row is opened from the batch gift entry table, GAU allocations will also be loaded in addition to the rest of the form.

# Issues Closed
[W-038511](https://foundation.lightning.force.com/lightning/r/agf__ADM_Work__c/a2x1E000001HqUjQAK/view)

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by dfuller114** [comment]
**lurch: attach W-038511

**Comment by dfuller114** [comment]
Reference PR #5130 to address previous comments

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HqUjQAK" target="_blank">W-038511</a>

File: src/classes/BGE_BatchGiftEntry_UTIL.cls [hunk 1]
```diff
@@ -88,6 +88,8 @@ public with sharing class BGE_BatchGiftEntry_UTIL {
 
         } else if (batch.Batch_Gift_Entry_Version__c >= 2.0) {
 
+            fieldApiNames.add(DataImport__c.Additional_Object_JSON__c.getDescribe().getName());
```
Comment (ptewson-sfdo) [inline src/classes/BGE_BatchGiftEntry_UTIL.cls]: Bringing over comment from Ben: "We don't have the ability to change this yet, but we should retrieve this field name from the Custom Settings Facade to future-proof this."

File: src/lwc/geFormWidgetAllocation/geFormWidgetAllocation.js [hunk 2]
```diff
@@ -152,21 +153,59 @@ export default class GeFormWidgetAllocation extends LightningElement {
...
+    @api
+    load(data) {
+        let dataImportRow = JSON.parse(data[DI_ADDITIONAL_OBJECT.fieldApiName]).dynamicSourceByObjMappingDevName;
+        if (!dataImportRow) {
+            return;
+        }
+        let rowList = new Array();
+        let fieldMappings = GeFormService.fieldMappings;
+        let gauMappingKeys = Object.keys(fieldMappings).filter(key => {
+            return key.toLowerCase().includes('gau_allocation_1');
```
Comment (ptewson-sfdo) [inline src/lwc/geFormWidgetAllocation/geFormWidgetAllocation.js]: Bringing over comment from Vlad: Can we add a constant for 'gau_allocation_1'.

**Comment by dfuller114** [comment]
closing this PR and re-parenting the branch to the `fast-follow` feature branch

**Pull request closed without merging**